lambdas/thumbnail/index.py

```diff
@@ -62,28 +188,24 @@ def lambda_handler(request):
 
     resp = requests.get(url)
     if resp.ok:
-        image_bytes = BytesIO(resp.content)
-        with Image.open(image_bytes) as image:
-            orig_format = image.format
-            orig_size = image.size
+        img = AICSImage(resp.content)
```

Comment:
@dimaryaz we're gonna have some huge responses here (300MB and up); not sure what we can do besides cranking RAM